addOns/accessControl/src/main/resources/org/zaproxy/zap/extension/accessControl/resources/Messages.properties

```diff
@@ -55,3 +55,12 @@ accessControl.alert.authorization.name = Access Control Issue - Improper Authori
 accessControl.alert.authorization.otherinfo = Accessed as user: {0}\n\nRequest detected as authorized: {1}. The defined access rule for resource is that access should be: {2}.
 accessControl.alert.authentication.name = Access Control Issue - Improper Authentication
 accessControl.alert.authentication.otherinfo = Accessed as an unauthenticated user.\n\nRequest detected as authorized: {0}. The defined access rule for resource is that access should be: {1}.
+
+accessControl.api.action.scan = Starts an Access Control scan with the given \
+context ID and user ID. (Optional parameters: user ID for Unauthenticated \
```

Comment:
- Worth noting that the "user ID" can be a comma separated list of IDs.
- For the unauthenticated user it's a boolean.
- Should mention the defaults?

Reply:
- [x] Worth noting that the "user ID" can be a comma separated list of IDs.
- [x] For the unauthenticated user it's a boolean.
- [ ] Should mention the defaults?